api/system-settings/getOtpToken

```diff
@@ -23,25 +23,34 @@
 use warnings;
 use strict;
 use JSON;
-use esmith::ConfigDB;
 use NethServer::ApiTools;
 use Convert::Base32;
 use NethServer::Password;
+use Sys::Hostname;
 
 my $db = esmith::ConfigDB->open_ro;
 my $input = NethServer::ApiTools::readInput();
 my $username = $input->{'username'} || '';
```

Comment:
Why the cockpit is calling this script directly?? From the browser console:
```
 echo '{"username":"giacomo"}' | /usr/libexec/nethserver/api/system-settings/getOtpToken | jq
```
This is against cockpit design: you should always call an api.

Also since the call is done without sudo, it doesn't make sense to read the user name from the stdin, just retrieve it from the script with something like:
```
my ($username,$passwd,$uid,$gid, $quota,$comment,$gcos,$home,$shell,$expire) = getpwuid( $< );

```